### ISSUE TITLE
Latest Admob (3.15.1 ) couses crashes on Launch (Google Play Games + Firebase Database )

### ISSUE THREAD

**Issue by studentutu** [googleads/googleads-mobile-unity#796]
Unity 2018.14f1, .Net 3.x, Android, build on Windows 10

1. Install Latest Google Play Games plugins for Unity (0.9.52)
2. Install Latest Firebase Database plugin (5.4.0) (No Errors occurs on Import)
(Try to build just in case anything else breaks -> Everything should be okay and App is working)
3. Install Google latest Admob Unity Plugin - 3.15.1 (No Errors occurs on Import)
Now : Try to build, Build succeeds but the application crashes right on Launch

Long Story Short:
Latest Admob Plugin somehow breaks the App and dependencies

**Comment by stewartmiles** [comment]
Did you do this https://developers.google.com/admob/unity/start#set_your_admob_app_id ?

**Comment by studentutu** [comment]
Yes! It was it!
Thank you!
